pkg/descheduler/evictions/evictions_test.go

```diff
@@ -40,35 +45,57 @@ func TestEvictPod(t *testing.T) {
 	tests := []struct {
 		description string
 		node        *v1.Node
-		pod         *v1.Pod
-		pods        []v1.Pod
-		want        error
+		evictedPod  *v1.Pod
+		pods        []runtime.Object
+		wantErr     bool
+		wantErrMsg  string
 	}{
 		{
 			description: "test pod eviction - pod present",
 			node:        node1,
-			pod:         pod1,
-			pods:        []v1.Pod{*pod1},
-			want:        nil,
+			evictedPod:  pod1,
+			pods:        []runtime.Object{pod1},
+			wantErr:     false,
```

Comment:
https://github.com/kubernetes-sigs/descheduler/pull/1536/files#r1837802730

we have other case like this

Reply:
Right. I missed this one. Me slowly disappearing ...